src/internal/pachd/builder.go

```diff
@@ -408,6 +408,22 @@ func (b *builder) initPachwController(ctx context.Context) error {
 	return nil
 }
 
+func (b *builder) startPFSWorker(ctx context.Context) error {
+	env, err := PFSWorkerEnv(b.env)
+	if err != nil {
+		return err
+	}
+	config := pfs_server.WorkerConfig{
+		Storage: b.env.Config().StorageConfiguration,
+	}
+	w, err := pfs_server.NewWorker(*env, config)
+	if err != nil {
+		return err
+	}
+	go w.Run(ctx) //nolint:errcheck
```

Comment:
Should we log the error like how you did it in testpachd?
```
go func() {
		if err := w.Run(ctx); err != nil {
			log.Error(ctx, "from worker", zap.Error(err))
		}
	}()
```

Reply:
Yeah you're probably right, will do.